data/strategies/StrategyAH2EarncyUSDT.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "strategy",
+  "name": "StrategyAH2EarncyUSDT",
+  "description": "Lends USDT on Alpha Homora v2 to generate interest. Users of Alpha Homora borrow USDT to perform leveraged yield-farming on Alpha Homora’s platform.",
```

Comment:
I haven't seen any reference to outside users in other descriptions. It might confuse some people